tests/testthat/test-modules.R

```diff
@@ -509,9 +509,35 @@ testthat::test_that("format.teal_modules returns proper structure", {
 
   appended_mods <- append_module(mods, mod3)
 
-  testthat::expect_equal(
-    gsub("\033\\[[0-9;]*m", "", format(appended_mods)),
-    "TEAL ROOT\n  |- a\n  |  |- Datasets         : all\n  |  |- Properties:\n  |  |  |- Bookmarkable  : FALSE\n  |  |  L- Reportable    : FALSE\n  |  |- UI Arguments     : \n  |  |- Server Arguments : \n  |  L- transformators       : \n  |- c\n  |  |- Datasets         : all\n  |  |- Properties:\n  |  |  |- Bookmarkable  : FALSE\n  |  |  L- Reportable    : FALSE\n  |  |- UI Arguments     : \n  |  |- Server Arguments : \n  |  L- transformators       : \n  L- c\n     |- Datasets         : all\n     |- Properties:\n     |  |- Bookmarkable  : FALSE\n     |  L- Reportable    : FALSE\n     |- UI Arguments     : \n     |- Server Arguments : \n     L- transformators       : \n" # nolint: line_length
+  testthat::expect_setequal(
```

Comment:
ℹ️ Fixes error and improves comparison as a long string is difficult to maintain.

@vedhav can you take a look if this is fine?

Reply:
Thanks! it works fine.